doc/modules/outlier_detection.rst

```diff
@@ -175,8 +176,54 @@ This strategy is illustrated below.
            Data Mining, 2008. ICDM'08. Eighth IEEE International Conference on.
 
 
-One-class SVM versus Elliptic Envelope versus Isolation Forest
---------------------------------------------------------------
+Local Outlier Factor
+--------------------
+Another efficient way to perform outlier detection on moderately high dimensional
+datasets is to use the Local Outlier Factor (LOF) algorithm.
+
+The :class:`neighbors.LocalOutlierFactor` (LOF) algorithm computes a score
+(called local outlier factor) reflecting the degree of abnormality of the
+observations.
+It measures the local density deviation of a given data point with respect to
+its neighbors. The idea is to detect the samples that have a substantially
+lower density than their neighbors.
```

Comment:
I'm just giving my own opinion here but I'm not sure we can say much on this except pointing to heuristics used in the original paper.
Tuning `n_neighbors` is as important (if less) as tuning the kernel bandwidth of the Gaussian kernel for the One-Class SVM. As far as I know, not much is said in scikit-learn on this issue for the One-Class SVM nor on the subsampling parameter or number of trees for Isolation Forest.